src/main/scala/com/decodified/scalassh/Command.scala

```diff
@@ -18,11 +18,26 @@ package com.decodified.scalassh
 
 import java.io.{ByteArrayInputStream, File, FileInputStream, InputStream}
 import net.schmizz.sshj.connection.channel.direct.Session
-
-final case class Command(command: String, input: CommandInput = CommandInput.NoInput, timeout: Option[Int] = None)
+trait Command {
+  val command: String
```

Comment:
All those `val`s should be defs.
Usually abstract members are defined as `def`s in Scala since this leaves the decision of how to implement the member to the "implementee", without any drawbacks.

Reply:
You're correct, but these are not longer here.